ci-operator/step-registry/openshift/microshift/e2e/metal-tests/openshift-microshift-e2e-metal-tests-commands.sh

```diff
@@ -168,12 +168,16 @@ EOF
 
 trap 'scp -r ${INSTANCE_PREFIX}:/home/${HOST_USER}/microshift/_output/test-images/scenario-info ${ARTIFACT_DIR}' EXIT
 
-# Run the scenario tests, if the phase script exists
-# (we can clean this up after the main PR lands)
-cd /microshift/test || true
+cat > /tmp/tests.sh << EOF
+cd ~/microshift/test
 if [ -f ./bin/ci_phase_test.sh ]; then
-    ./bin/ci_phase_test.sh
+  ./bin/ci_phase_test.sh
```

Comment:
Do you think we could remove the `if [ -f ./bin/ci_phase_test.sh ]; then`? Maybe in another PR?
The file is in microshift's main, so it's not needed